packages/react-components/src/components/MessageThread.tsx

```diff
@@ -966,6 +973,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
             showMessageStatus={showMessageStatus}
             messageStatus={messageProps.message.status}
             onActionButtonClick={onActionButtonClickMemo}
+            dateTimeFormat={dateTimeFormat}
```

Comment:
As `dateTimeFormat` is an optional prop, don't we need to add a check if prop is available or not.